test/parallel/test-runner-cli.js

```diff
@@ -269,10 +269,22 @@ const testFixtures = fixtures.path('test-runner');
 
 {
   // --test-shard option, first shard
+  const allShardsTestsFiles = [
+    'a.cjs',
+    'b.cjs',
+    'c.cjs',
+    'd.cjs',
+    'e.cjs',
+    'f.cjs',
+    'g.cjs',
+    'h.cjs',
+    'i.cjs',
+    'j.cjs',
+  ].map((file) => join(testFixtures, 'shards', file));
```

Comment:
perhaps we should `readDir` and iterate the files instead of hardcoding these?

Reply:
done